policy/modules/contrib/glusterd.te

```diff
@@ -79,6 +79,7 @@ allow glusterd_t self:tcp_socket { accept listen };
 allow glusterd_t self:unix_stream_socket { accept listen connectto };
 allow glusterd_t self:rawip_socket create_socket_perms;
 allow glusterd_t self:unix_stream_socket create_stream_socket_perms;
+allow glusterd_t self:netlink_rdma_socket { bind create setopt };
```

Comment:
Please use one of the macros available for socket permissions in https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/support/obj_perm_sets.spt